lib/api/bucketPutVersioning.js

```diff
@@ -16,6 +16,46 @@ import services from '../services';
     x-amz-mfa: [SerialNumber] [TokenCode]
  */
 
+
+function waterfall1ValidateAuthorization(mdValParams, log, next) {
+    services.metadataValidateAuthorization(mdValParams, (err, bucket) => {
+        if (err) {
+            log.trace('request authorization failed', {
```

Comment:
There's already logs for that, I don't think this is necessary. Especially since that would just get rid of that function since you could then use `services.metadataValidateAuthorization()` directly in the waterfall.